presto-benchmark/src/main/java/com/facebook/presto/benchmark/CountAggregationBenchmark.java

```diff
@@ -46,7 +46,7 @@ protected List<? extends OperatorFactory> createOperatorFactories()
         JavaAggregationFunctionImplementation countFunction = functionAndTypeManager.getJavaAggregateFunctionImplementation(
                 functionAndTypeManager.lookupFunction("count", fromTypes(BIGINT)));
         AggregationOperatorFactory aggregationOperator = new AggregationOperatorFactory(1, new PlanNodeId("test"),
-                Step.SINGLE, ImmutableList.of(AggregationUtils.bind(countFunction, ImmutableList.of(0), Optional.empty())), false);
+                Step.SINGLE, ImmutableList.of(GenericAccumulatorFactory.generateAccumulatorFactory(countFunction, ImmutableList.of(0), Optional.empty())), false);
```

Comment:
nits: we prefer static import as long as the function name is not confusing, so here static import `GenericAccumulatorFactory.generateAccumulatorFactory` and this can just be `ImmutableList.of(generateAccumulatorFactory(countFunction, ImmutableList.of(0), Optional.empty()))`

Reply:
Please fix all other places to static import `GenericAccumulatorFactory.generateAccumulatorFactory` as well.

Reply:
👌